odo/backends/tests/test_postgres.py

```diff
@@ -91,8 +92,24 @@ def complex_sql(url):
     except sa.exc.OperationalError as e:
         pytest.skip(str(e))
     else:
-        yield t
-        drop(t)
+        try:
+            yield t
+        finally:
+            drop(t)
+
+
+@pytest.yield_fixture
+def sql_with_array(url):
+    ds = 'var * {a: var * float64, b: string}'
+    try:
+        t = resource(url, dshape=ds)
```

Comment:
do you think we should explicitly create the sql table here. If there is a bug where this doesn't actually create an array but is also not discovered as an array then the test will pass.